ql/termstructures/volatility/equityfx/fixedlocalvolsurface.cpp

```diff
@@ -132,7 +132,8 @@ namespace QuantLib {
 
         for (Size i=0; i < strikes_.size(); ++i)
             for (Size j=1; j<strikes_[i]->size(); j++) {
-                QL_REQUIRE(strikes_[i]->at(j)>=strikes_[i]->at(j-1),
+                QL_REQUIRE(strikes_[i]->at(j)>=strikes_[i]->at(j-1)
+                    || close_enough(strikes_[i]->at(j),strikes_[i]->at(j-1)),
```

Comment:
What happens in the calculations if two strikes are decreasing but close enough?  Do they still make sense, or should the case be detected and treated?

Reply:
Depends on the interpolation method being used. But the problem can also be solved in the HestonSLVFDMModel directly.